docs/source/version-5-development.md

```diff
@@ -101,3 +101,20 @@ npm run docz -- dev
 ```
 
 After compilation has finished you can access docz at http://localhost:3030/.
+
+
+## Contributing a Translation
```

Comment:
I would instead suggest linking to the https://github.com/coralproject/talk/blob/master/CONTRIBUTING.md#localization page as that's where we update when code changes. Feel free to back port some of this text over to that contributing guidelines!

Reply:
I will add that link as well, but not going to remove this from the docs page as it does not hurt to have it here as well.